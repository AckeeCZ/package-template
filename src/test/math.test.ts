import sum from 'lib/math';

describe('Math', () => {
    test('sum', () => {
        expect(sum(1, 2)).toEqual(3);
    });
    test('sum with calculation', () => {
        expect(sum(5, 3, true)).toMatchSnapshot();
    });
});
