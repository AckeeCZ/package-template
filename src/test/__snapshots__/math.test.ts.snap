// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Math sum with calculation 1`] = `
Object {
  "calculation": "5 + 3 = 8",
  "sum": 8,
}
`;
